experimenter/experimenter/nimbus-ui/src/components/PageHome/DirectoryTable/index.tsx

```diff
@@ -145,18 +144,18 @@ export const DirectoryColumnEndDate: ColumnComponent = ({
 );
 
 export const DirectoryColumnUnpublishedUpdates: ColumnComponent = ({
-  publishStatus: p,
+  isRolloutDirty: d,
 }) => (
   <td data-testid="directory-table-cell">
-    {p ? (
+    {d ? (
       <>
         <Badge
           className={
             "ml-2 border rounded-pill px-2 bg-white font-weight-normal border-danger text-danger"
           }
           data-testid="directory-unpublished-updates"
         >
-          {p === NimbusExperimentPublishStatusEnum.DIRTY ? "YES" : ""}
+          {d === true ? "YES" : ""}
```

Comment:
You can just get rid of `=== true` altogether.

Reply:
Also we're checking `d` above so it will always be `true` by the time we get down here?